### PR TITLE
Moved e2e to a scripted test

### PR DESCRIPTION
**Pull request by javierarrieta**
Moved the end to end tests to a scripted test so it is more consistent with plugin testing in sbt projects

### CODE REVIEW

**Comment by javierarrieta** [comment]
Hmm, some failures, investigating...

File: e2e/project/project/Version.scala [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Generated by Avro Codegen's build.sbt.
```
Comment (pkinsky) [inline e2e/project/project/Version.scala]: Does this need to be checked in?

Comment (javierarrieta) [inline e2e/project/project/Version.scala]: Well spotted, thanks

**Comment by malcolmgreaves** [comment]
This looks good to me. @javierarrieta want to merge it in? (double check version numbers and such, following semantic versioning etc.)

**Comment by javierarrieta** [comment]
Any reason this was closed? @malcolmgreaves what do you mean about version numbers?

**Comment by pkinsky** [comment]
@bparli @javierarrieta @malcolmgreaves not sure why this was closed, I had meant to merge it earlier but I've been out most of this week with the flu

**Comment by malcolmgreaves** [comment]
@bparli closed it... no idea why. Reopen it if you need to (are you able to?)

**Comment by malcolmgreaves** [comment]
@javierarrieta the comment about version numbers doesn't apply here, I think. I'm just stating that version numbers are *not* automatically created based upon commits. Bumping version numbers is only important for releases. So new features, refactoring, or patches. This PR is all test code, so I think it's totally fine to leave the version numbers as is.

**Comment by javierarrieta** [comment]
Ok, now it shows a different history in the branch, maybe something happened to master, I will try to cherrypick the changes and apply to a new branch

**Comment by malcolmgreaves** [comment]
@javierarrieta something weird happened with the repo history... it was reverted to a really, really, really old version. I had to force push the last known good, public release. This might mess up your PR. I suggest opening up a new one so we can cleanly merge it going forward.

**Comment by bparli** [comment]
Was this being worked on in two places (i.e. gihub enterprise and also here?).  It sounds like this version was overwritten in the migration

**Pull request closed without merging**